src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/BottomTabBar.tsx

```diff
@@ -75,6 +76,21 @@ function BottomTabBar({selectedTab}: BottomTabBarProps) {
         Navigation.navigate(route);
     }, [activeWorkspaceID, selectedTab]);
 
+    const navigateToSearch = useCallback(() => {
+        if (selectedTab === SCREENS.SEARCH.BOTTOM_TAB) {
+            return;
+        }
+        interceptAnonymousUser(() => {
+            const currentSearchParams = getCurrentSearchParams();
```

Comment:
Oops, it seems getCurrentSearchParams has never been used in our app

Reply:
It was used in search but we are still working on this feature. I think the guys refactored the current state of search so it's not used but it's rather a temporary situation.